src/lib/invalidTiming.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const invalidTiming = (timing: number) => {
```

Comment:
```suggestion
export const isInvalidTimestamp = (timing: DOMHighResTimeStamp) => {
```

Rationale:
* I think starting with "is" in the name makes it more clear in the code that this function returns a boolean.
* Also, since you're checking the validity of a `DOMHighResTimeStamp` instance, I think it's better to use that term rather than "timing".

...and then make the corresponding changes throughout this PR.

Reply:
Done in https://github.com/GoogleChrome/web-vitals/pull/440/commits/4372eb20162b13986761cccecdfb72836fe81f25

I also changed the `timing` argument name to `timestamp` and renamed the file.